docs/resources/rds_account.md

```diff
@@ -0,0 +1,52 @@
+---
+subcategory: "Relational Database Service (RDS)"
+---
+
+# huaweicloud_rds_account
+
+Manages RDS account resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "instance_id" {}
+
+resource "huaweicloud_rds_account" "test" {
+  instance_id = var.instance_id
+  name        = "test"
+  password    = "Test@12345678"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the rds account resource. If omitted, the
+  provider-level region will be used. Changing this creates a new resource.
+
+* `instance_id` - (Required, String, ForceNew) Specifies the rds instance id. Changing this will create a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the username of the db account. The username consists of 1 to 32
```

Comment:
how about remove 'The username consists of 1 to 32' as there is detailed description bellow

Reply:
done